src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V>[] table;
+    private int threshold;
```

Comment:
Actually, you can omit it and just simply refer to DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR.